app/jobs/export_dsi_users_to_big_query_job.rb

```diff
@@ -1,13 +1,10 @@
-require "export_dsi_users_to_big_query"
-require "export_dsi_approvers_to_big_query"
-
 class ExportDSIUsersToBigQueryJob < ApplicationJob
   queue_as :low
 
   def perform
     return if DisableExpensiveJobs.enabled?
 
-    ExportDSIUsersToBigQuery.new.run!
-    ExportDSIApproversToBigQuery.new.run!
+    Publishers::DfeSignIn::BigQueryExport::Users.new.call
+    Publishers::DfeSignIn::BigQueryExport::Approvers.new.call
```

Comment:
- Used modules/classes to structure and namespace the code. 
- Normalised the service objects public interface to `.call`